.babelrc

```diff
@@ -4,19 +4,7 @@
     "@babel/preset-react"
   ],
   "plugins": [
-    "@babel/plugin-syntax-dynamic-import",
-    "@babel/plugin-syntax-import-meta",
-    "@babel/plugin-proposal-class-properties",
-    "@babel/plugin-proposal-json-strings",
-    [
-      "@babel/plugin-proposal-decorators",
-      {
-        "legacy": true
-      }
-    ],
-    "@babel/plugin-proposal-function-sent",
-    "@babel/plugin-proposal-export-namespace-from",
-    "@babel/plugin-proposal-numeric-separator",
-    "@babel/plugin-proposal-throw-expressions"
+    "@babel/plugin-transform-runtime",
+    "@babel/plugin-proposal-class-properties"
```

Comment:
`@babel/plugin-proposal-class-properties` is actually included in [`@babel/preset-env@7.12.1`](https://github.com/babel/babel/blob/v7.12.1/packages/babel-preset-env/package.json#L24) and can be removed as well!

Reply:
I tried removing it but am getting an error locally - I even tried updating to the latest @babel/preset-env but it still seems to require that plugin...

Reply:
Never mind, you were right - they just need to update their error message, as it still says you need to install that plugin when in reality you just need to pass an option to preset-env - thanks!